bazel 0.27.0 no workey

Current versions:

```
$ bazel version
Bazelisk version: v0.0.7
Build label: 0.27.0
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Mon Jun 17 13:00:46 2019 (1560776446)
Build timestamp: 1560776446
Build timestamp as int: 1560776446
```

```
$ git log -n 1
commit 8a5b5d7f861283ea93ac19ae68655fa758e438a4 (HEAD -> master, upstream/master)
Merge: af3a7104d be5779675
Author: Kubernetes Prow Robot <k8s-ci-robot@users.noreply.github.com>
Date:   Thu Jun 27 08:53:24 2019 -0700

    Merge pull request #13173 from oxddr/enable-kube-proxy-scraping1
    
    Enable scraping kube-proxy in ci-kubernetes-e2e-gci-gce-scalability.
```

Output:

```
$ bazel clean --expunge
INFO: Starting clean (this may take a while). Consider using --async if the clean takes more than several minutes.
$ bazel test //prow/...
Starting local Bazel server and connecting to it...
DEBUG: /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:256:5: rbe_default is using checked-in configs 'struct(config_repos = [], create_cc_configs = True, create_java_configs = True, env = {"ABI_LIBC_VERSION": "glibc_2.19", "ABI_VERSION": "clang", "BAZEL_COMPILER": "clang", "BAZEL_HOST_SYSTEM": "i686-unknown-linux-gnu", "BAZEL_TARGET_CPU": "k8", "BAZEL_TARGET_LIBC": "glibc_2.19", "BAZEL_TARGET_SYSTEM": "x86_64-unknown-linux-gnu", "CC": "clang", "CC_TOOLCHAIN_NAME": "linux_gnu_x86"}, java_home = "/usr/lib/jvm/java-8-openjdk-amd64", name = "9.0.0")'
INFO: Analyzed 118 targets (780 packages loaded, 12740 targets configured).
INFO: Found 118 test targets...
ERROR: /home/skuznets/fakego/src/k8s.io/test-infra/prow/cmd/deck/static/BUILD.bazel:7:1: Compiling TypeScript (devmode) //prow/cmd/deck/static:api failed (Exit 1)
error TS2688: Cannot find type definition file for 'color'.
error TS2688: Cannot find type definition file for 'color-convert'.
error TS2688: Cannot find type definition file for 'color-name'.
error TS2688: Cannot find type definition file for 'google.visualization'.
error TS2688: Cannot find type definition file for 'gtag.js'.
error TS2688: Cannot find type definition file for 'jasmine'.
error TS2688: Cannot find type definition file for 'long'.
error TS2688: Cannot find type definition file for 'node'.
error TS2688: Cannot find type definition file for 'pako'.


INFO: Elapsed time: 106.877s, Critical Path: 2.86s
INFO: 108 processes: 108 linux-sandbox.
FAILED: Build did NOT complete successfully
```

/cc @fejta @Katharine @cjwagner

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -27,7 +27,7 @@ git_repository(
 
 load("@bazel_skylib//lib:versions.bzl", "versions")
 
-versions.check(minimum_bazel_version = "0.26.0")
+versions.check(minimum_bazel_version = "0.27.0")
 
 # TODO(fejta): delete the com_google_protobuf rule once rules_go >= 0.18.4
 # This silences the shallow_since message (and increases cache hit rate)
@@ -156,8 +156,8 @@ git_repository(
 # https://github.com/bazelbuild/rules_nodejs
 http_archive(
     name = "build_bazel_rules_nodejs",
-    sha256 = "e04a82a72146bfbca2d0575947daa60fda1878c8d3a3afe868a8ec39a6b968bb",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.31.1/rules_nodejs-0.31.1.tar.gz"],
+    sha256 = "6d4edbf28ff6720aedf5f97f9b9a7679401bf7fca9d14a0fff80f644a99992b4",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.32.2/rules_nodejs-0.32.2.tar.gz"],
 )
 
 load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")

--- a/hack/BUILD.bazel
+++ b/hack/BUILD.bazel
@@ -37,7 +37,7 @@ nodejs_test(
         "//:all-srcs",
         "@npm//tslint",
     ],
-    entry_point = "@npm//node_modules/tslint:bin/tslint",
+    entry_point = "@npm//:node_modules/tslint/bin/tslint",
     tags = ["lint"],
     templated_args = [
         "-p",

--- a/package.json
+++ b/package.json
@@ -6,9 +6,9 @@
     "pako": "^1.0.10"
   },
   "devDependencies": {
-    "@bazel/bazel": "0.26.1",
-    "@bazel/jasmine": "0.31.1",
-    "@bazel/typescript": "0.31.1",
+    "@bazel/bazel": "0.27.0",
+    "@bazel/jasmine": "0.32.2",
+    "@bazel/typescript": "0.32.2",
     "@types/color": "^3.0.0",
     "@types/google.visualization": "^0.0.43",
     "@types/gtag.js": "^0.0.0",

